docs/resources/ekscluster.md

```diff
@@ -283,25 +283,25 @@ Optional:
 
 Optional:
 
-- `vpc_cni_config` (Block List, Max: 1) VPC CNI addon config contains the configuration for the VPC CNI addon of the cluster (see [below for nested schema](#nestedblock--spec--config--addons_config--vpc_cni_config))
+- `vpc_cni_config` (Block List, Max: 1) VPC CNI addon config contains the configuration for the VPC CNI addon of the cluster. (see [below for nested schema](#nestedblock--spec--config--addons_config--vpc_cni_config))
 
 <a id="nestedblock--spec--config--addons_config--vpc_cni_config"></a>
 ### Nested Schema for `spec.config.addons_config.vpc_cni_config`
 
 Optional:
 
-- `eni_config` (Block List) ENI config for the VPC CNI addon (see [below for nested schema](#nestedblock--spec--config--addons_config--vpc_cni_config--eni_config))
+- `eni_config` (Block List) ENI config is the VPC CNI Elastic Network Interface config for providing the configuration of subnet and security groups for pods in each AZ.  with subnet as a key and list of security group as value. The subnets are expected to be in different AZs. Nodepools will be created only in the az where the pod subnets resides. (see [below for nested schema](#nestedblock--spec--config--addons_config--vpc_cni_config--eni_config))
```

Comment:
`with subnet as a key and list of security group as value.`
Is Subnet a key or just a member of ENIConfig?
`The subnets are expected to be in different AZs. Nodepools will be created only in the az where the pod subnets resides.`
Let's not use the term pod subnet as we don't use this term anywhere else. Does this sound better? `Nodepool subnets need to be in the same AZ as the AZs used in ENIConfig`

Reply:
@DheerajSShetty fixed this